pkg/server/server.go

```diff
@@ -56,6 +56,13 @@ var expectedFilesDuringInitialization = map[string]struct{}{
 	"tuning.yaml":    {},
 }
 
+const (
```

Comment:
Can we not use snake_case in Go? Let's be uniform with the rest and use (lowercase)CamelCase for (private)constants.